src/sage/env.py

```diff
@@ -245,7 +266,11 @@ def var(key: str, *fallbacks: Optional[str], force: bool = False) -> Optional[st
 # GAP memory and args
 
 SAGE_GAP_MEMORY = var('SAGE_GAP_MEMORY', None)
-_gap_cmd = "gap -r"
+
+# Passing -A allows us to use a minimal GAP installation without
+# producing errors at start-up. The files sage.g and sage.gaprc can be
+# used to load any additional packages that may be available.
+_gap_cmd  = f'gap -A -l "{GAP_ROOT_PATHS}"'
```

Comment:
Can we move this logic of determining the fallback of the gap command to `interfaces/gap_workspace.py`? It feels like that there is more context why this default command is the correct choice.

Reply:
I think this is here so it can be overriden by environment `SAGE_GAP_COMMAND`

Reply:
Right, the whole chunk is,

```python
_gap_cmd  = f'gap -A -l "{GAP_ROOT_PATHS}"'
if SAGE_GAP_MEMORY is not None:
    _gap_cmd += " -s " + SAGE_GAP_MEMORY + " -o " + SAGE_GAP_MEMORY
SAGE_GAP_COMMAND = var('SAGE_GAP_COMMAND', _gap_cmd)
```

So `_gap_cmd` is being used as the default for `SAGE_GAP_COMMAND`. (And thus needs to be set before that call to `var`).

Reply:
Maybe it's reasonable to have here just:
```
SAGE_GAP_COMMAND = var("SAGE_GAP_COMMAND")
```
and move the logic for the "default" gap command to `interfaces/gap_workspace.py` as something like:
```
gap_cmd = SAGE_GAP_COMMAND
if gap_cmd is None:
    gap_cmd = f'gap -A -l "{GAP_ROOT_PATHS}"'
    if SAGE_GAP_MEMORY is not None:
        gap_cmd += " -s " + SAGE_GAP_MEMORY + " -o " + SAGE_GAP_MEMORY
```

Reply:
Yes, thanks @tornaria, that's what I had in mind.

Reply:
Ok, done. Although I wound up moving it to `sage.interfaces.gap` (and not `sage.interface.gap_workspace`) because that's where most of the existing gap command processing takes place.